Tests/DataTable/Column/ColumnTest.php

```diff
@@ -3,12 +3,13 @@
 
 
 use CrossKnowledge\DataTableBundle\DataTable\Table\Element\Column\Column;
+use PHPUnit\Framework\TestCase;
 
-class ColumnTest extends \PHPUnit_Framework_TestCase
+class ColumnTest extends TestCase
 {
     public function testUndefinedOptionThrowsException()
     {
-        $this->setExpectedException('Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException');
+        $this->expectException('Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException');
```

Comment:
```suggestion
        $this->expectException(UndefinedOptionsException::class);
```
Is it possible if we declare this class in the uses block and use ::class here?

Reply:
Done.